### DISCUSSION TITLE
Is it possible to improve launch time ?

### DISCUSSION THREAD

**Discussion by SenkiJun** [Jelmerro/Vieb#552] [Q&A]
Thank you so much for this amazing browser, it's just perfect.  However the launch time is significantly longer compared to other browsers on windows 11.
Is it possible to improve this via launching on start up like a daemon or with any other method ?

**Comment by SenkiJun** [comment]
I noticed normal version launches very fast so I am closing this. (I was using the portable version before)

**Comment by Jelmerro** [comment]
Sorry, for the late response, which portable version are we talking about here? And please note that this has been worked on before in the following issue, and has been closed as most things possible to improve speedup are already done: https://github.com/Jelmerro/Vieb/issues/66

**Reply by SenkiJun** [reply]
Portable x64 (exe)     <-- This one was slow.

Portable x64 (zip)    <-- This one launches super fast as the other browsers.

Thank you so much for creating Vieb. It's just perfect. 
(Also sorry didn't see that issue thread.)

**Reply by Jelmerro** [reply]
This is probably because the standalone exe needs to extract itself on startup I figure, but I am not directly responsible for this part of the program, as I'm using electron-builder for this, but I can imagine this being the reason for the speed difference.

**Reply by SenkiJun** [reply]
Got it, you're probably right. All the other versions are working great. Thanks again for dedicating your time to this awesome project.
